packages/espresso/android/build.gradle

```diff
@@ -79,8 +79,18 @@ dependencies {
 
     // Assertions
     api 'androidx.test.ext:junit:1.1.5'
-    api 'androidx.test.ext:truth:1.5.0'
-    api 'com.google.truth:truth:1.1.3'
+    // Don't make these 'api', otherwise another workaround for guava 32+ on gradle 6.x is needed:
+    // 1.
+    // dependencies.constraints {
+    //     api("com.google.guava:guava") {
+    //         attributes {
+    //             attribute(Attribute.of("org.gradle.jvm.environment", String), "android")
+    //         }
+    //     }
+    // }
+    // 2. make api dependency on guava instead implementation
+    implementation 'androidx.test.ext:truth:1.5.0'
+    implementation 'com.google.truth:truth:1.1.3'
```

Comment:
issue with espresso was because workaround https://github.com/flutter/packages/pull/6535/commits/a3800a16550faf8ce7a1c56a893883fafe635901 works for `implementation` configuration, not `api`
and for it to work with `api` one needs `api` dependency on guava as well (which I think is worse solution)

Reply:
dropped commit